api/v1/openapi.yaml

```diff
@@ -664,6 +664,9 @@ definitions:
       policy:
         description: Policy information of endpoint
         "$ref": "#/definitions/EndpointPolicy"
+      policy-enabled:
```

Comment:
What do you think about moving this into `EndpointPolicy` as a member `enabled`?